index.js

```diff
@@ -96,8 +97,14 @@ const getMudletSnapshotLinksForPr = async prNumber => {
         return -1
       }
     }))
-    // now take the latest link
-    .mapValues(value => value[0])
+    // for windows, take the two latest links: windows-64 and windows-32
+    .mapValues((value, platform) => {
+      if (platform === 'windows') {
+        return value.filter(val => /windows-64|windows-32/.test(val.filename)).slice(0, 2)
```

Comment:
These may end up to be for the same architecture in certain cases, but let's wait and see if this can happen at all.